internal/server/v1/firehose/handlers.go

```diff
@@ -443,6 +460,45 @@ func getFirehoseResource(ctx context.Context, client entropyv1beta1.ResourceServ
 	return mapResourceToFirehose(resp.GetResource(), false)
 }
 
+func getFirehoseHistory(ctx context.Context, client entropyv1beta1.ResourceServiceClient, firehoseURN string) ([]revisionDiff, error) {
+	resp, err := client.GetResourceRevisions(ctx, &entropyv1beta1.GetResourceRevisionsRequest{Urn: firehoseURN})
+	if err != nil {
+		st := status.Convert(err)
+		if st.Code() == codes.NotFound {
+			return nil, errors.ErrNotFound.
+				WithMsgf(firehoseNotFound).
+				WithCausef(st.Message())
+		}
+		return nil, err
+	}
+
+	prevSpec := []byte{'{', '}'}
```

Comment:
You can write this as `[]byte("{}")` as well btw.

Reply:
Done!